### DISCUSSION TITLE
Add staffmsg to %cmi_user_toggle_*% placeholder

### DISCUSSION THREAD

**Discussion by PovilasLT** [Zrips/CMI#6952] [Enhancement]
Hello!
In our server, we use /staffmsg a lot. But when it's toggleable, we often send messages in wrong chats :D 
If placeholder %cmi_user_toggle_staffmsg% will be added, we could use this placeholder to display staffmsg status in scoreboard :)

**Comment by mrfloris** [comment]
That's a pretty fun idea, upvoted
